lib/polyfill/media_capabilities.js

```diff
@@ -37,10 +37,15 @@ shaka.polyfill.MediaCapabilities = class {
     // See: https://github.com/shaka-project/shaka-player/issues/3582
     // TODO: re-evaluate MediaCapabilities in the future versions of PS5
     // Browsers.
+    // Since MediaCapabilities is not fully supported on some Firefox yet,
```

Comment:
What specific parts of MediaCapabilities are not supported on Firefox?  It might be useful to have that in a comment, too, so we can follow up in the future.  Is there a bugzilla issue filed about it?  That could also be useful.

Reply:
I just read #4292.  I see that you're trying to fix compatibility with very very old Firefox versions (72).  Why is that your target version?

If you need to do this specifically for very old versions, you should figure out which ones and add a version check.  I don't think we should force-install a polyfill on a current, working, and supported browser version.

Reply:
I think so...